### PR TITLE
add device support for rejia HR109 and orvibo RC808ZB

### PR DESCRIPTION
**Pull request by fantasyjqc**

### CODE REVIEW

**Comment by Koenkk** [comment]
For HR109, do you have a picture with white background?

**Comment by fantasyjqc** [comment]
> 



IMG_1737

File: src/devices/orvibo.js [hunk 1]
```diff
@@ -56,6 +84,15 @@ module.exports = [
             'button_3_click', 'button_3_hold', 'button_3_release', 'button_4_click', 'button_4_hold', 'button_4_release'])],
         toZigbee: [],
     },
+    {
+        zigbeeModel: ['RemoteControl-22'],
+        model: 'RC808ZB',
+        vendor: 'ORVIBO',
+        description: 'ORVIBO Remote Control',
+        fromZigbee: [fzLocal.orvibo_raw_3, fz.battery],
+        exposes: [e.battery()],
```
Comment (Koenkk) [inline src/devices/orvibo.js:93]: I notice that `e.action([])` is missing here, please add all the possible actions here.

Example: https://github.com/Koenkk/zigbee-herdsman-converters/pull/5786/files#diff-d81231be627ae3c846eecc8731c4130661783a89b5d05f855658e9755d1b994bR83

File: src/converters/fromZigbee.js [hunk 2]
```diff
@@ -6746,6 +6746,13 @@ const converters = {
             return {action: 'vibration'};
         },
     },
+    REJIA_vibration: {
+        cluster: 'ssIasZone',
+        type: 'commandStatusChangeNotification',
+        convert: (model, msg, publish, options, meta) => {
+            return {action: true};
```
Comment (Koenkk) [inline src/converters/fromZigbee.js:6753]: `true` should be `'vibration'` here (to match `e.action(['vibration'])`)

**Pull request closed without merging**